### ISSUE TITLE
error in page editing

### ISSUE THREAD

**Issue by zafarnayab** [joomdev/JD-Builder#26]
When the **JD page builder's** page is opened for editing a page, it displays an  **Page not found (404) Error** 

<img width="1680" alt="Screenshot 2020-05-15 at 2 24 16 AM" src="https://user-images.githubusercontent.com/19945423/81986186-ffb9b000-9654-11ea-84c8-bb29b1c65c08.png">

**Comment by chetanmadaan** [comment]
This is a known issue and happens in a few cases.

To get around, link the page to a menu item and try editing again.

**Comment by zafarnayab** [comment]
This page is already linked with the main menu and works fine in frontend. This error occur while I'm trying to modify the page.

**Comment by Nishtha003** [comment]
Was the page assigned to any other menu item that is unpublished now, please confirm this because if it was earlier assigned to other menu item and now to another it creates menu itemid conflict with the one that was assigned first!

**Comment by zafarnayab** [comment]
No, the page assigned only with the main menu and it is published.

**Comment by zafarnayab** [comment]
Any Update?

**Comment by chandanndeep** [comment]
Hello @zafarnayab,

Please share the "**Open in New Window**" link.
![image](https://user-images.githubusercontent.com/17069784/82524044-84239b80-9b4b-11ea-99ad-fd84fbd33c80.png)


Thanks,
Chandan
